Update AWS FSX procedure (#2731)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-fsx.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Contenu du pack

### Modèles

Le connecteur de supervision **Amazon FSx** apporte un modèle d'hôte :

* **Cloud-Aws-Fsx-custom**

Le connecteur apporte les modèles de service suivants
(classés selon le modèle d'hôte auquel ils sont rattachés) :

<Tabs groupId="sync">
<TabItem value="Cloud-Aws-Fsx-custom" label="Cloud-Aws-Fsx-custom">

| Alias         | Modèle de service                  | Description                                                |
|:--------------|:-----------------------------------|:-----------------------------------------------------------|
| Fsx-DataUsage | Cloud-Aws-Fsx-Datausage-Api-custom | Contrôle les métriques liées au système de fichier FSx     |
| Fsx-Freespace | Cloud-Aws-Fsx-Freespace-Api-custom | Contrôle l'espace disponible sur un système de fichier FSx |

> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Cloud-Aws-Fsx-custom** est utilisé.

</TabItem>
</Tabs>

### Règles de découverte

#### Découverte d'hôtes

> | Nom de la règle | Description             |
> |:----------------|:------------------------|
> | Amazon AWS FSX  | Discover Amazon AWS FSX |

Author: ```suggestion
| Nom de la règle | Description             |
|:----------------|:------------------------|
| Amazon AWS FSX  | Discover Amazon AWS FSX instances |
```

Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.

### Métriques & statuts collectés

Voici le tableau des services pour ce connecteur, détaillant les métriques rattachées à chaque service.

<Tabs groupId="sync">
<TabItem value="Fsx-DataUsage" label="Fsx-DataUsage">

| Métrique                | Unité |
|:------------------------|:------|
| fsx.data.read.bytes     | B     |
| fsx.data.write.bytes    | B     |
| fsx.data.io.read.count  | count |
| fsx.data.io.write.count | count |
| fsx.metadata.ops.bytes  | B     |

</TabItem>
<TabItem value="Fsx-Freespace" label="Fsx-Freespace">

| Métrique             | Unité |
|:---------------------|:------|

> | fsx.storage.free.byt | B     |

Author: ça m'a paru bizarre, mais c'était déjà comme ça avant...

| Macro                    | Description                                                                                                                    | Valeur par défaut | Obligatoire |
|:-------------------------|:-------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| STATISTIC                | Set the metric calculation method (Only Sum is relevant)                                                                       | sum               |             |
| TIMEFRAME                | Set timeframe in seconds                                                                                                       | 900               |             |
| PERIOD                   | Set period in seconds                                                                                                          | 60                |             |
| FILTERMETRIC             | Filter on a specific metric Can be: DataReadBytes, DataWriteBytes, DataReadOperations, DataWriteOperations, MetaDataOperations |                   |             |

> | WARNINGDATAREADBYTES     |                                                                                                                                |                   |             |
> | CRITICALDATAREADBYTES    |                                                                                                                                |                   |             |
> | WARNINGDATAREADOPS       |                                                                                                                                |                   |             |
> | CRITICALDATAREADOPS      |                                                                                                                                |                   |             |
> | WARNINGDATAWRITEOPS      |                                                                                                                                |                   |             |
> | CRITICALDATAWRITEOPS     |                                                                                                                                |                   |             |
> | WARNINGMETADATAOPSBYTES  |                                                                                                                                |                   |             |
> | CRITICALMETADATAOPSBYTES |                                                                                                                                |                   |             |

Author: ```suggestion
| WARNINGDATAREADBYTES     | Thresholds                                                                                                                     |                   |             |
| CRITICALDATAREADBYTES    | Thresholds                                                                                                                     |                   |             |
| WARNINGDATAREADOPS       | Thresholds                                                                                                                     |                   |             |
| CRITICALDATAREADOPS      | Thresholds                                                                                                                     |                   |             |
| WARNINGDATAWRITEOPS      | Thresholds                                                                                                                     |                   |             |
| CRITICALDATAWRITEOPS     | Thresholds                                                                                                                     |                   |             |
| WARNINGMETADATAOPSBYTES  | Thresholds                                                                                                                     |                   |             |
| CRITICALMETADATAOPSBYTES | Thresholds                                                                                                                     |                   |             |
```

| EXTRAOPTIONS             | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                            | --verbose         |             |

</TabItem>
<TabItem value="Fsx-Freespace" label="Fsx-Freespace">

| Macro                    | Description                                                                                         | Valeur par défaut | Obligatoire |
|:-------------------------|:----------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| STATISTIC                | Set the metric calculation method (Default: Average). Can be 'minimum', 'average'                    | average           |             |

> | TIMEFRAME                |                                                                                                     | 900               |             |
> | PERIOD                   |                                                                                                     | 60                |             |
> | FILTERMETRIC             |                                                                                                     |                   |             |

Author: ```suggestion
| TIMEFRAME                | Set timeframe in seconds (e.g. '3600' to check last 60 minutes) | 900               |             |
| PERIOD                   | Set period in seconds.                                                             | 60                |             |
| FILTERMETRIC             | Name of the metrics to filter on.                                       |                   |             |
```

| --command           | Command to get information (Default: 'aws'). Can be changed if you have output in a file.                                                                                                                                             |
| --command-path      | Command path (Default: none).                                                                                                                                                                                                         |
| --command-options   | Command options (Default: none). Only use for testing purpose, when you want to set ALL parameters of a command by yourself.                                                                                                          |
| --proxyurl          | Proxy URL if any                                                                                                                                                                                                                      |

> | --skip-ssl-check    | Avoid certificate issuer verification. Useful when AWS resources are hosted by a third-party.  Note that it strips all stderr from the command result. Will be enhanced someday. Debug will only display CLI instead of evreything.   |

Reviewer: Will be enhanced someday!!!

Author: Why not ¯\_(ツ)_/¯

| --period            | Set period in seconds.                                                                           |
| --timeframe         | Set timeframe in seconds.                                                                        |
| --statistic         | Set cloudwatch statistics (Can be: 'minimum', 'maximum', 'average', 'sum').                      |
| --zeroed            | Set metrics value to 0 if none. Useful when CloudWatch does not return value when not defined.   |
| --proxyurl          | Proxy URL if any                                                                                 |

</TabItem>
</Tabs>

#### Options des modes

Les options disponibles pour chaque modèle de services sont listées ci-dessous :

<Tabs groupId="sync">
<TabItem value="Fsx-DataUsage" label="Fsx-DataUsage">

| Option              | Description                                                                                                                             |
|:--------------------|:----------------------------------------------------------------------------------------------------------------------------------------|

> | --name              | Set the instance name (Required) (can be defined multipletimes).                                                                        |

Reviewer: ```suggestion
| --name              | Set the instance name (required) (can be defined multiple times).                                                                        |
```

> | --filter-metric     | Filter on a specific metric Can be: DataReadBytes, DataWriteBytes, DataReadOperations, DataWriteOperations, MetaDataOperations          |

Reviewer: ```suggestion
| --filter-metric     | Filter on a specific metric. Can be: DataReadBytes, DataWriteBytes, DataReadOperations, DataWriteOperations, MetaDataOperations          |
```

| --warning-$metric$  | Thresholds warning ($metric$ can be: 'data-write-ops', 'data-write-ops', 'data-read-ops', 'data-read-bytes', 'metadata-ops-bytes').     |
| --critical-$metric$ | Thresholds critical ($metric$ can be: 'data-write-ops', 'data-write-ops', 'data-read-ops', 'data-read-bytes', 'metadata-ops-bytes').    |

</TabItem>
<TabItem value="Fsx-Freespace" label="Fsx-Freespace">

| Option                        | Description                                                        |
|:------------------------------|:-------------------------------------------------------------------|

> | --name                        | Set the instance name (Required) (can be defined multipletimes).   |

Reviewer: ```suggestion
| --name                        | Set the instance name (required) (can be defined multiple times).   |
```

> | --warning-storage-free-bytes  | Thresholds warning on remaining available bytes.                   |

Reviewer: ```suggestion
| --warning-storage-free-bytes  | Warning threshold for remaining available bytes.                   |
```

> | --critical-storage-free-bytes | Thresholds warning on remaining available bytes.                   |

Reviewer: ```suggestion
| --critical-storage-free-bytes | Warning threshold for remaining available bytes.                   |
```
